en/docs/install-and-setup/setup/sso/configuring-external-idp-using-oidc-for-multi-tenancy.md

```
@@ -0,0 +1,367 @@
+    === "On Linux/Mac OS"
+        ```
+        sh wso2server.sh
+        ```
+
+## Configure Identity Server
+
+### Step 1 - Create New Organization
+
+1.  Sign in to WSO2 IS Management Console:
+
+    ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)